doc/source/wsrep-provider-index.rst

```diff
@@ -432,11 +432,29 @@ size of the disk. Pages are prefixed by gcache.page.
 
 .. seealso::
 
-   |galera| Documentation: gcache.page_size
+   Galera Documentation: gcache.page_size
       https://galeracluster.com/library/documentation/galera-parameters.html#gcache-page-size
-   |percona| Database Performance Blog: All You Need to Know About GCache (Galera-Cache)
+   Percona Database Performance Blog: All You Need to Know About GCache (Galera-Cache)
       https://www.percona.com/blog/2016/11/16/all-you-need-to-know-about-gcache-galera-cache/
 
+.. variable:: gcache.recover
+
+   :cli: No
+   :conf: Yes
+   :scope: Global
+   :dyn: No
+   :default: No
+
+Attempts to recover a node's gcache file to a usable state on startup. If the node can successfully
+recover the gcache file, the node can provide IST to the remaining nodes. This ability can 
```

Comment:
I would decipher the abbreviation (or at least create the :abbr: directive to give the full name on hover)

Reply:
Noted.